test/src/tests-cookie-syncing.js

```diff
@@ -39,7 +65,7 @@ describe('cookie syncing', function() {
             isProduction: true,
             settings: {},
             frequencyCap: 14,
-            pixelUrl: pixelUrl,
+            pixelUrl: '',
```

Comment:
I think this `pixelSettings` object is the same everywhere. Would it make sense to promote this to the top of the file?